src/commands/simbrief.ts

```diff
@@ -0,0 +1,13 @@
+import { CommandDefinition } from '../lib/command';
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+export const simbrief: CommandDefinition = {
+    name: 'simbrief',
+    description: 'Provides a link to the FlyByWire Simbrief integration tutorial video.',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | Simbrief Integration',
+        description: 'Watch a tutorial on how to use the FlyByWire A32NX Simbrief Integration [here](https://youtu.be/l_5J7ZuEEtk)',
```

Comment:
Also suggest adding a footer to this command to establish that the SB username is now entered in the flyPad not MCDU, and explaining the W+B page is different but functions the same, this is all expected to move to the flyPad so wording in a way that pre-empts that change would be best.

Reply:
Same here. No need for another simbrief command